config/GameUserSettings.ini

```diff
@@ -30,6 +30,18 @@ TamingSpeedMultiplier=2.00000
 XPMultiplier=2.00000
 DayTimeSpeedScale=0.80000
 NightTimeSpeedScale=1.20000
+PerLevelStatsMultiplier_Player[0]=1.5
+PerLevelStatsMultiplier_Player[1]=1.2
+PerLevelStatsMultiplier_Player[2]=1.5
+PerLevelStatsMultiplier_Player[3]=1.5
+PerLevelStatsMultiplier_Player[4]=1.2
+PerLevelStatsMultiplier_Player[5]=1.2
+PerLevelStatsMultiplier_Player[6]=1.5
+PerLevelStatsMultiplier_Player[7]=1.5
+PerLevelStatsMultiplier_Player[8]=1.5
+PerLevelStatsMultiplier_Player[9]=2
+PerLevelStatsMultiplier_Player[10]=2
+PerLevelStatsMultiplier_Player[11]=2
```

Comment:
Seems those values needs to be in  the Game.ini and not this file : 
https://wiki.nitrado.net/en/Per-level_stat_multipliers_using_the_Game.ini_file_in_ARK:_Survival_Evolved